lib/stdlib/src/proc_lib.erl

```diff
@@ -596,31 +622,26 @@ my_info(Class, Reason, StartF, Stacktrace) ->
      my_info_1(Class, Reason, Stacktrace)].
 
 my_info_1(Class, Reason, Stacktrace) ->
+    Keys = [registered_name, dictionary, message_queue_len,
+            links, trap_exit, status, heap_size, stack_size, reductions],
+    PInfo = get_process_info(self(), Keys),
+    {dictionary, Dict} = lists:keyfind(dictionary,1,PInfo),
     [{pid, self()},
-     get_process_info(self(), registered_name),         
+     lists:keyfind(registered_name,1,PInfo),
+     {process_description, get_process_description(self())},
```

Comment:
waste of characters and also readers eyeballs to me. what about having `{tag, get_tag()}`? or `{id, get_id()}` instead? would it be shorter and equally informative?

Reply:
I'm using the API function here?